### PR TITLE
removes k8s.io/kubernetes/pkg/api dependency from the webhook plugin.

### PR DESCRIPTION
**Pull request by p0lyn0mial**
**What this PR does / why we need it**: removes `k8s.io/kubernetes/pkg/api` dependency from `webhook` plugin. The runtime.scheme can be injected to the webhook from the plugin initializer.

**Release note**:

```
NONE
```

### CODE REVIEW

**Comment by p0lyn0mial** [comment]
how convenient :)

**Comment by p0lyn0mial** [comment]
@deads2k FYI

File: plugin/pkg/admission/webhook/admission.go [hunk 1]
```diff
@@ -41,7 +41,6 @@ import (
 	genericadmissioninit "k8s.io/apiserver/pkg/admission/initializer"
 	clientset "k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
-	"k8s.io/kubernetes/pkg/api"
```
Comment (p0lyn0mial) [inline plugin/pkg/admission/webhook/admission.go]: that's the essence.

**Comment by caesarxuchao** [comment]
lgtm. @deads2k @sttts what do you think?

File: plugin/pkg/admission/webhook/admission.go [hunk 2]
```diff
@@ -130,6 +126,15 @@ func (a *GenericAdmissionWebhook) SetServiceResolver(sr admissioninit.ServiceRes
 	}
 }
 
+// SetScheme sets a serializer(NegotiatedSerializer) which is derived from the scheme
+func (a *GenericAdmissionWebhook) SetScheme(scheme *runtime.Scheme) {
+	if scheme != nil {
```
Comment (sttts) [inline plugin/pkg/admission/webhook/admission.go]: shouldn't this panic for nil?

**Comment by deads2k** [comment]
remaining packages are the "tail", approving

**Pull request merged**